templates/miq-template-ext-db.yaml

```diff
@@ -490,6 +490,16 @@ objects:
         ProxyPassReverse / http://${NAME}/
       </VirtualHost>
 - apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: "${HTTPD_SERVICE_NAME}-auth-configs"
+  data:
+    auth-type: internal
+    auth-configuration.conf: |
+      # External Authentication Configuration File
+      #
+      # For details on usage please see https://github.com/ManageIQ/manageiq-pods/blob/master/README.md
```

Comment:
You can actually link directly to your new section with : 

`https://github.com/ManageIQ/manageiq-pods/blob/master/README.md#configuring-external-authentication`